setup.py

```diff
@@ -44,7 +44,8 @@
         "pymeshlab~=2022.2",
         "click~=8.1.3",
         "aenum~=3.1.15",
-        "rtree~=1.2.0"
+        "rtree~=1.2.0",
+        "telemoma~=0.1.2"
```

Comment:
I think we need to remove this - I don't like the idea of having this kind of dependency on a new, unproven module. We can discuss with the team on Slack.